pandas/conftest.py

```diff
@@ -953,3 +954,76 @@ def __len__(self):
             return self._data.__len__()
 
     return TestNonDictMapping
+
+
+indices_dict = {
+    "unicode": tm.makeUnicodeIndex(100),
+    "string": tm.makeStringIndex(100),
+    "datetime": tm.makeDateIndex(100),
+    "datetime-tz": tm.makeDateIndex(100, tz="US/Pacific"),
+    "period": tm.makePeriodIndex(100),
+    "timedelta": tm.makeTimedeltaIndex(100),
+    "int": tm.makeIntIndex(100),
+    "uint": tm.makeUIntIndex(100),
+    "range": tm.makeRangeIndex(100),
+    "float": tm.makeFloatIndex(100),
+    "bool": tm.makeBoolIndex(2),
+    "categorical": tm.makeCategoricalIndex(100),
+    "interval": tm.makeIntervalIndex(100),
+    "empty": Index([]),
+    "tuples": MultiIndex.from_tuples(zip(["foo", "bar", "baz"], [1, 2, 3])),
+    "repeats": Index([0, 0, 1, 1, 2, 2]),
+}
+
+
+@pytest.fixture(params=indices_dict.keys())
```

Comment:
this needs to be `sorted(indices_dict.keys())` or else running the test suite will break with multiprocessing

Reply:
Dictionaries are sorted since 3.6, so the order should already be deterministic (all tests on CI succeeded every time, even after pushing multiple times)

Reply:
The `indices` fixture is part of the test suite for quite some time already. I just moved the location as suggested by jreback. I really think this is fine.